rosidl_runtime_c/include/rosidl_runtime_c/string_functions.h

```diff
@@ -142,6 +160,24 @@ rosidl_runtime_c__String__Sequence__are_equal(
   const rosidl_runtime_c__String__Sequence * lhs,
   const rosidl_runtime_c__String__Sequence * rhs);
 
+/// Copy rosidl_runtime_c__String__Sequence structure content.
+/**
+ * This functions performs a deep copy, as opposed to the shallow copy
+ * that plain assignment yields.
+ *
+ * \param[out] input a pointer to a rosidl_runtime_c__String__Sequence
```

Comment:
```suggestion
 * \param[in] input a pointer to a rosidl_runtime_c__String__Sequence
```

Reply:
Fixed in https://github.com/ros2/rosidl/pull/650/commits/ab79f66aac8c5e876481596bb9cb06eb6a1e6e3b.